src/components/tag/tag.hbs

```diff
@@ -1,3 +1,12 @@
+{{#if componentsX}}
+<ul role="listbox">
+  {{#each tags}}
+  <li class="bx--tag bx--tag--{{type}}" tabindex="0">{{label}}</li>
+  {{/each}}
```

Comment:
If these are clickable, we should use `a` or `button` elements, which will receive focus without having to use `tabindex`